README.md

```diff
@@ -22,3 +23,53 @@ Where
 - If omitted, the link will apply to all products in the directory.
 - Test path is relative to the current directory, so will typically be just a filename.
 - Test result status is a status as defined in the wpt.fyi codebase.
+
+## How to contribute to wpt-metadata repository
+If you are a memeber of the web-platform-tests org, you can contribute to the wpt-metadata repo by sending out a PR directly or through a UI-based approach on wpt.fyi. We also have a Triage Metadata API available for trusted third parties
```

Comment:
I don't think this is correct:

- Can't anyone send a PR to this repository? Don't need to be a member of wpt org
- We don't have a UI yet, let's not mention it